src/app/hearing-part/components/listing-create/listing-create.component.html

```diff
@@ -10,11 +10,11 @@
                 <div class="error" *ngIf="errors">Error: {{errors}}</div>
 
                 <mat-form-field>
-                    <input id="caseNumber" formControlName="caseNumber" [(ngModel)]="listing.hearingPart.caseNumber" matInput placeholder="Case number *" type="text">
+                    <input id="caseNumber" formControlName="caseNumber" [(ngModel)]="listing.hearingPart.caseNumber" maxlength="{{caseNumberMaxLength}}" matInput placeholder="Case number *" type="text">
```

Comment:
Kamil Zieliński mentioned  that maxLength validation should be done as part of ReactiveForm definition. All validation rules should be inside FormGroup. Same to caseTitle

Reply:
discussed with Kamil, and finally I've added validator to FormGroup